docs/REST API/rest-api.md

```diff
@@ -4591,6 +4592,28 @@ For example, a policy might ask “Is Gatekeeper enabled on macOS devices?“ Th
 }
 ```
 
+---
+
+### Count Policies
+
+`GET /api/v1/fleet/global/policies/count`
+
```

Comment:
Can we add parameters?

#### Parameters

| Name                    | Type    | In    | Description                                                                                                                                                                                                                                                                                                                                 |
| ----------------------- | ------- | ----- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| query                   | string  | query | Search query keywords. Searchable fields include `name`.  |